presto-docs/src/main/sphinx/static/versions.json

```diff
@@ -0,0 +1,9 @@
+{
+    "0.244": "0.244",
+    "0.243.3": "0.243.3",
+    "0.243.2": "0.243.2",
+    "0.243.1": "0.243.1",
+    "0.243": "0.243",
+    "0.242.1": "0.242.1",
+    "0.242": "0.242",
+}
```

Comment:
What is the plan for this file? Are we going to require people to update it for each release?

Reply:
I removed that file; it was unused.